src/libcore/python/plugin_v.cpp

```diff
@@ -1,13 +1,98 @@
 #include <mitsuba/core/plugin.h>
 #include <mitsuba/core/properties.h>
+#include <mitsuba/core/spectrum.h>
+#include <mitsuba/core/transform.h>
 #include <mitsuba/python/python.h>
 
+void parse_spectrum(PluginManager &pgmr, Properties &prop, py::dict &dict) {
```

Comment:
As a general comment, it would be great to use 4 spaces for indentation so to be consistent with the rest of the codebase 😉